kafka-client/src/main/java/dev/responsive/kafka/api/ResponsiveStores.java

```diff
@@ -241,10 +245,7 @@ public static <K, V> StoreBuilder<TimestampedWindowStore<K, V>> timestampedWindo
   public static <K, V> Materialized<K, V, KeyValueStore<Bytes, byte[]>> materialized(
       final String name
```

Comment:
Should we change this to `ResponsiveKeyValueParams`? Since it's not directly analogous to any Streams APIs and no one is using it yet, I think we can just change it rather than adding a new overload. Your call though

Reply:
I will go ahead and change it 👍